upgrading/topics/rhsso/intro.adoc

```diff
@@ -14,7 +14,7 @@ RH-SSO instances from a single control point. The upgrade process differs depend
 implemented. Specific instructions for each mode are provided where applicable.
 
 The purpose of this guide is to document the steps that are required to successfully upgrade from
-Red Hat Single Sign-On 7.2 to Red Hat Single Sign-On 7.3.
+Red Hat Single Sign-On 7.2 or Keycloak 4.8.x to Red Hat Single Sign-On 7.3.
```

Comment:
I would remove this. It's sufficient with the separate section on migrating from Keycloak.